job_spec/INSAR_ISCE_TEST.yml

```diff
@@ -27,6 +27,13 @@ INSAR_ISCE_TEST:
           minLength: 67
           maxLength: 67
           example: S1B_IW_SLC__1SDV_20210711T014947_20210711T015013_027740_034F80_D404
+    esd_coherence_threshold:
+      api_schema:
+        description: Coherence threshold value, between 0 and 1, for points to include when performing ESD, or -1 for no ESD correction.
```

Comment:
```suggestion
        description: Coherence threshold value for determining which points to include when performing ESD. Must be any value between 0 and 1 (inclusive), or -1 for no ESD correction.
```

Reply:
Yep, I prefer this wording. Thanks @jtherrmann